tests/tests-api.php

```diff
@@ -169,6 +169,8 @@ public static function wpSetUpBeforeClass() {
 	public function setUp() {
 		parent::setUp();
 
+		wp_set_current_user( self::$user_id );
```

Comment:
In the parent `tearDown()` function, the current user was being reset to `0` by WordPress core. This meant that all the `EDD()->api->update_key( self::$user_id );` calls were failing, due to `current_user_can()` returning false.